sygnal/http.py

```diff
@@ -203,15 +203,15 @@ async def cb():
                 root_span.finish()
 
     def find_pushkins(self, appid):
-        """
-        Finds matching pushkins in self.sygnal.pushkins according to the appid.
-        If it finds a specific pushkin with the exact app id, immediately returns
-        it (as a list).
-        Otherwise returns possible pushkins (as a list).
+        """Finds matching pushkins in self.sygnal.pushkins according to the appid.
 
-        appid: app identifier to search in self.sygnal.pushkins.
+        Args:
+            appid (str): app identifier to search in self.sygnal.pushkins.
+        Returns:
```

Comment:
```suggestion

        Args:
            appid (str): app identifier to search in self.sygnal.pushkins.

        Returns:
```

(sections should be separated by blank lines)